src/Geometry.h

```diff
@@ -76,21 +76,28 @@ class Geometry {
 
 private:
 
-  /** The boundary conditions at the top of the bounding box containing
-   *  the Geometry. False is for vacuum and true is for reflective BCs. */
-  boundaryType _top_bc;
+  /** The boundary conditions at the x-min surface of the bounding box 
+   * containing the Geometry. False is for vacuum and true is for reflective 
```

Comment:
Is the second sentence here about "False" vs. "true" for vacuum/reflective BCs needed? I know this was here prior to this PR, but upon a second look at this, it seems that this is unneeded since these attributes are of type `boundaryType` rather than booleans.